src/pages/Dongwu/Main/Asides/Feed/Photo/PhotoComment/Comments/CommentTemplate.module.scss

```
@@ -0,0 +1,60 @@
+@mixin flex-set(
+  $flex-direction: column,
+  $justify-content: center,
+  $align-items: center
+) {
+  display: flex;
+  flex-direction: $flex-direction;
+  justify-content: $justify-content;
+  align-items: $align-items;
+}
+
+.commentsDesc {
+  @include flex-set(row, initial, initial);
+  position: relative;
+  height: 18px;
+  line-height: 1.5rem;
+  font-size: 14px;
+
+  > div {
+    &:first-child {
+      font-weight: 700;
+      margin-right: 5px;
+      > a {
+        text-decoration: none;
+        color: black;
+      }
+    }
+  }
+
+  > div {
+    &:last-child {
+      font-weight: 400;
```

```suggestion
  > div {
    &:first-child {
      font-weight: 700;
      margin-right: 5px;
      > a {
        text-decoration: none;
        color: black;
      }
    }
  }


  &:last-child {
    font-weight: 400;
```
이렇게 묶어줄 수 있을거 같아요. 그리고 `>` 선택자는 순서에 의존하기 때문에 스타일이 있다면 가급적 클래스 셀렉터 사용해서 스타일링 해주는게 좋습니다.

---

새롭게 안 사실인 것 같습니다. 앞으로는 공식문서의 내용을 보다 꼼꼼히 읽어보도록 하겠습니다.